bfast.ts

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line no-use-before-define
+export { BFastHeader, BFast }
```

Comment:
I've typically seen `export` statements at the bottom of files once things have been declared. Has the trend reversed?